### PR TITLE
Fix dark icon colors for five/vapor under time slider above flag

### PR DESCRIPTION
**Pull request by madikarizma**
Also, fix bug preventing inset control bar overrides under time slider above flag

JW7-3805

### CODE REVIEW

File: src/css/skins/five.less [hunk 1]
```diff
@@ -133,4 +131,24 @@
         border-radius: @five-border-radius;
     }
 
+    // override control bar element colors that don't look good in time slider above mode
+    &.jw-flag-time-slider-above,
+    &.jw-breakpoint-0,
+    &.jw-breakpoint-1 {
+      .jw-controlbar {
+        .jw-text {
```
Comment (egreaves) [inline src/css/skins/five.less]: Couldn't we just set the `@inactive-color` and `@hover-color` vars for `.jw-flag-time-slider-above` etc for both skins?

Comment (madikarizma) [inline src/css/skins/five.less]: @egreaves The goal here was to only override those colors for control bar elements and when in time slider above flag/small player size. We don't want to override the colors anywhere else. However, there is a cleaner way I could do this.. thinking about it.

Comment (robwalch) [inline src/css/skins/five.less]: What's wrong with custom styling in a skin file?

Comment (egreaves) [inline src/css/skins/five.less]: @robwalch nothing's wrong with it. I was asking if we could update the color var, which should have been a comment, not a review, but after talking with @madikarizma I understand why what I was thinking isn't possible. I'm ok with merging this.

**Pull request merged**